internal/controller/pkg/manager/reconciler.go

```diff
@@ -392,6 +423,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	pr.SetControllerConfigRef(p.GetControllerConfigRef())
 	pr.SetWebhookTLSSecretName(r.webhookTLSSecretName)
 	pr.SetESSTLSSecretName(r.essTLSSecretName)
+	sn := fmt.Sprintf(fmtTLSServerSecretName, p.GetName())
+	cn := fmt.Sprintf(fmtTLSClientSecretName, p.GetName())
```

Comment:
Nit: can you make sure the resulting string is not longer than 64 chars and cut the provider name in case it is?

Reply:
Thanks for reminding that @phisco. It's 253 chars for [resource names](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#:~:text=This%20means%20the%20name%20must,alphanumeric%20characters%2C%20'%2D'%20or%20'.). Updated accordingly.